phase1/openstack/Kconfig

```diff
@@ -0,0 +1,53 @@
+menu "Openstack configuration"
+
+config phase1.openstack.os_image
+	string "OS image for nodes"
+	default "ubuntu-16.04"
+	help
+	  The OS image to use for deploying vm.
+
+config phase1.openstack.flavor_name
+	string "flavor name"
+	default "m1.medium"
+	help
+	  The flavor to use for the vm instance.
+
+config phase1.openstack.ssh_key_file
+	string "path to ssh private key file"
+	default "~/.ssh/id_rsa.terraform"
+	help
+	  The path to ssh private key file with which to configure
+	  the vm for auto-login. The public key file should also
+	  exist at the same path prefixed by .pub.
+
+config phase1.openstack.project
+	string "Openstack tenant"
```

Comment:
I think `tenant` is the term mostly used in OpenStack, So instead of `project` i used the `tenant` term everywhere now.
Also it is tenant name and not the 'ID'. So mentioned that too in the help message.